model/role.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose')
+
```

Comment:
you need to import claim schema here because you referenced the schema

Reply:
eg const Claim = require("./claim")

Reply:
Oh, I see. 👍

Reply:
Thanks